### DISCUSSION TITLE
Vits Loss Help

### DISCUSSION THREAD

**Discussion by SyGen899** [coqui-ai/TTS#2396] [General Q&A]
Hi Everyone, I was hoping I could get some help with interpreting the vits graphs, fairly new to this and wracking my brain on it for awhile, but from training so far the test samples sound great to me, but my concern is that it might be overfitting, Im training with roughly 10k samples totalling roughly 5 hours of data. any insights here would be appreciated.

![training graph](https://user-images.githubusercontent.com/79766385/223389809-550b19d4-d7e2-421e-86a7-549d462bb768.png)

**Comment by ramonrovirosa** [comment]
@SyGen899 did you figure out how to interpret the loss & if your graph is as expected?
